subgraphs/aave-forks/src/helpers.ts

```diff
@@ -132,6 +135,60 @@ export function getCollateralBalance(market: Market, account: Address): BigInt {
   return balanceResult.value;
 }
 
+export function getPrincipal(
+  market: Market,
+  account: Address,
+  side: string,
+  interestRateType: InterestRateType | null = null
+): BigInt | null {
```

Comment:
nit: early returns are generally a better pattern

https://methodpoet.com/return-early-or-if-statement/#:~:text=As%20a%20general%20rule%2C%20the,to%20understand%20and%20work%20with.